cypress/integration/clients_test.spec.ts

```diff
@@ -350,6 +355,197 @@ describe("Clients test", () => {
     });
   });
 
+  describe("Roles tab test", () => {
+    const rolesTab = new ClientRolesTab();
+    let client: string;
+
+    before(() => {
+      keycloakBefore();
+      loginPage.logIn();
+      sidebarPage.goToClients();
+
+      client = "client_" + (Math.random() + 1).toString(36).substring(7);
+
+      listingPage.goToCreateItem();
+
+      createClientPage
+        .selectClientType("openid-connect")
+        .fillClientData(client)
+        .continue()
+        .save();
+      masthead.checkNotificationMessage("Client created successfully", true);
+    });
+
+    beforeEach(() => {
+      keycloakBefore();
+      loginPage.logIn();
+      sidebarPage.goToClients();
+      listingPage.searchItem(client).goToItemDetails(client);
+      rolesTab.goToRolesTab();
+    });
+
+    after(() => {
+      adminClient.deleteClient(client);
+    });
+
+    it("should fail to create client role with empty name", () => {
```

Comment:
It could be better visually to have the first word of all the test phrases capitalized
```suggestion
    it("Should fail to create client role with empty name", () => {
```

Reply:
Good idea @Aboullos - fixed